Automattic/wp-calypso
# Help Center: make auth and status timeouts shorter

#### Proposed Changes

* Currently, we keep the chat availability status cached for the entire session. And availability is fluctuant and can easily change during the session. This PR makes it so that we check every 10 mins when the user is not looking at the contact us page in the help center, and 1 minute while they are. This is important to avoid starting a chat when there are no active operators. This will mean that there will be a brief 500-600ms socket connection to check status every 10 minutes.
* It also changes the auth validity to 10 mins, by auth I mean fetching the JWT token for Happychat auth.

#### Testing instructions
1. Open the Help Center and go to the contact page.
2. There should be no chat availability.
3. Go to https://hud-staging.happychat.io/ and set yourself as available.
4. Wait 1 minute, the chat status should update automatically to available without refreshing.
